run-servers.sh

```diff
@@ -0,0 +1,148 @@
+fi
+
+
+SERVER_VERSION="10.1.2.Final"
+SERVER_HOME=server/infinispan-server-$SERVER_VERSION
+CLUSTER_SIZE_MAIN="$SERVER_HOME/bin/cli.sh -c localhost:11322 -f batch "
+ZIP_ROOT="http://downloads.jboss.org/infinispan"
+
+CONF_DIR_TO_COPY_FROM="spec/configs/"
+IS_SSL_PROCESSED=0
+SERVER_DIR="infinispan-js-serverng"
+
+function waitForClusters()
+{
+cat > batch<<EOF
+describe
+disconnect
+EOF
+
+  MEMBERS_MAIN=''
+  while [ "$MEMBERS_MAIN" != '3' ];
+  do
+    MEMBERS_MAIN=$($CLUSTER_SIZE_MAIN | grep cluster_size | cut -d':' -f2 | sed 's/.$//' | sed -e 's/^[[:space:]]*//')
+    echo "Waiting for clusters to form (main: $MEMBERS_MAIN)"
+    sleep 20
+  done
+}
+
+function prepareServerDir()
+{
+    local isCi=$1
+    local confPath=$2
+    local isSsl=$3
+    local dirName=${4}
+
+    if [ ! -f server/infinispan-server-$SERVER_VERSION.zip ]; then
+        cd server
+        wget $ZIP_ROOT/$SERVER_VERSION/infinispan-server-$SERVER_VERSION.zip
+        unzip  infinispan-server-$SERVER_VERSION.zip
+        cd ..
+    fi
+
+    if [[ ${isCi} = "--ci" ]]; then
+        SERVER_TMP=${SERVER_HOME}
+        echo "Use server in directory: $SERVER_TMP"
+    else
+        if [[ -z "${SERVER_TMP}" ]]; then
+             SERVER_TMP=${TMPDIR:-'/tmp'}/${SERVER_DIR}
+             mkdir ${SERVER_TMP} 2>/dev/null
+             echo "Created temporary directory: $SERVER_TMP"
+
+             cp -r ${SERVER_HOME}/* $SERVER_TMP
+             echo "Server copied to temporary directory."
+
+             $SERVER_TMP/bin/user-tool.sh -u admin -p 'mypassword'
+             echo "Admin user added."
+        fi
+
+        cp -r ${SERVER_HOME}/server ${SERVER_TMP}/${dirName}
+    fi
+
+
+    cp "${CONF_DIR_TO_COPY_FROM}/${confPath}" ${SERVER_TMP}/${dirName}/conf
+    echo "Infinispan configuration file ${confPath} copied to server ${dirName}."
+
+    if [[ ${isSsl} = "true" && ${IS_SSL_PROCESSED} = 0 ]]; then
+        ./make-ssl.sh
+        echo "Generate TLS/SSL certificates"
+
+        IS_SSL_PROCESSED=1
+    fi
+
+    if [[ ${isSsl} = "true" ]]; then
+        cp out/ssl/ca/ca.jks $SERVER_TMP/${dirName}/conf
+        cp out/ssl/server/server.jks $SERVER_TMP/${dirName}/conf
+        cp out/ssl/sni-trust1/trust1.jks $SERVER_TMP/${dirName}/conf
+        cp out/ssl/sni-trust2/trust2.jks $SERVER_TMP/${dirName}/conf
+        cp out/ssl/sni-untrust/untrust.jks $SERVER_TMP/${dirName}/conf
+        echo "Security key and trust stores copied to temporary server."
+    fi
+    export SERVER_TMP=${SERVER_TMP}
+}
+
+function startServer()
+{
+    local isCi=$1
+    local confPath=$2
+    local isSsl=$3
+    local port=${4}
+    local nodeName=${5}
+    local jvmParam=${6}
+
+    prepareServerDir "${isCi}" ${confPath} ${isSsl} ${nodeName}
+
+    if [[ ! -z ${port} ]]; then
+        portStr="-p ${port}"
+    fi
+
+
+    if [[ ${isCi} = "--ci" ]]; then
+      nohup $SERVER_TMP/bin/server.sh -c ${confPath} "${portStr:-""}" --node-name=${nodeName} ${jvmParam:-} &
```

Comment:
`-s` options missing and remove `"` between `${portStr:-""}`
`nohup $SERVER_TMP/bin/server.sh -c ${confPath} -s ${SERVER_TMP}/${nodeName} ${portStr:-""} --node-name=${nodeName} ${jvmParam:-} &`